base_layer/wallet/src/output_manager_service/service.rs

```diff
@@ -551,7 +554,27 @@ where
         let mut shutdown = self.resources.shutdown_signal.clone();
         let mut base_node_watch = self.resources.connectivity.get_current_base_node_watcher();
         let event_publisher = self.resources.event_publisher.clone();
+        let validation_in_progress = self.validation_in_progress.clone();
         tokio::spawn(async move {
+            // Note: We do not want the validation task to be queued
+            let mut _lock = match validation_in_progress.try_lock() {
+                Ok(val) => val,
+                _ => {
+                    if let Err(e) = event_publisher.send(Arc::new(OutputManagerEvent::TxoValidationInternalFailure(id)))
```

Comment:
This wll fail a lot because the validation task has been designed to run for multiple outputs. We should use a channel and a long running service if we want one to run at a time (like in #4626 )

Reply:
But if this works then I think we see and perhaps implement that later if needed